cmd/plaid.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"github.com/quesurifn/go-digit/banker"
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultPublicToken string = "replaceme"
+)
+
+// Flags
+var (
+	publicToken string
+)
+
+var plaidCmd = &cobra.Command{
+	Use:   "plaid",
+	Short: "Wraps useful plaid commands",
+	Long:  "Wraps useful plaid commands",
+}
+
+var plaidExchangeTokenCmd = &cobra.Command{
+	Use:   "exchange",
+	Short: "Exchanges a public token for an access one",
+	Long:  "Exchanges a public token for an access one. Use the --token flag to specify the token",
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg := &banker.PlaidConfig{
+		// Stuff here
```

Comment:
Make sure you add the following... I'll break this out into something that will read from the config file or environment variables later:

```go
&banker.PlaidConfig{
  ClientID: "clientidgoeshere",
  Secret: "sekretz",
  PublicKey: "publickey",
}
```